src/core/packages/ui-settings/server-internal/src/saved_objects/migrations.ts

```diff
@@ -228,4 +228,25 @@ export const migrations = {
     }),
     references: doc.references || [],
   }),
+  '8.19.0': (doc: SavedObjectUnsanitizedDoc<any>): SavedObjectSanitizedDoc<any> => ({
```

Comment:
This is the legacy mechanism for adding migrations. You should use the model version interface. See line 17 in this file.

CI is also failing an early check:

```
Migration for type config for version 8.19.0 registered after switchToModelVersionAt (8.10.0)
```